q2_moshpit/plugin_setup.py

```diff
@@ -1802,6 +1802,28 @@
     citations=[]
 )
 
+TMR = TypeMatch([Kraken2Reports % Properties('reads'),
+                 Kraken2Reports % Properties('contigs')])
+TMO = TypeMatch([Kraken2Outputs % Properties('reads'),
+                 Kraken2Outputs % Properties('contigs')])
+plugin.methods.register_function(
+    function=q2_moshpit.kraken2.classification.filter_kraken2_classifications,
+    inputs={
+        'reports': SampleData[TMR],
+        'outputs': SampleData[TMO]
+    },
+    parameters={
+        'abundance_threshold': Float % Range(0, 100, inclusive_end=True)},
+    outputs=[('filtered_reports', SampleData[TMR]),
+             ('filtered_outputs', SampleData[TMO])],
+    input_descriptions={},
+    parameter_descriptions={},
+    output_descriptions={},
+    name='Filter Kraken2 Classifications by Abundance',
+    description='...',
```

Comment:
Should we just put here the text from the "name" field or did you want to keep the `...`?